### DISCUSSION TITLE
Multiple electric outages

### DISCUSSION THREAD

**Discussion by LParkhill** [NREL/REopt-API-Analysis#42] [4 - Resilience Questions] [answered]
Can the REopt web tool model multiple electric outages in one year, as might be seen in an area prone to wildfires?

**Comment by LParkhill** [comment]
Currently the REopt® web tool can only model one outage per year. The team has plans for enhancing the outage modeling capabilities.

**Reply by Galgallog3** [reply]
Is this still the case? Thank you.

**Comment by kurt-verdant** [comment, marked as answer]
My recollection is I've entered as many as 4 outages per year in the Web tool. It's been awhile though and I don't recall all the details. But here's a picture of where I did it.
<img width="1408" height="897" alt="image" src="https://github.com/user-attachments/assets/d84acf87-b5c8-4b9d-93a8-0f6c75c32737" />
